saleor/order/migrations/0139_fix_undiscounted_total_on_lines.py

```diff
@@ -0,0 +1,115 @@
+from functools import partial
+
+from django.db import migrations
+from django.db.models import F, Q
+from django.db.models.signals import post_migrate
+from saleor.plugins.manager import get_plugins_manager
+from saleor.order.app import OrderAppConfig
+
+# We need to sent `order_updated` for each updated order. When we drop plugin manager
+# we should rewrite `send_order_updated`
+
+BATCH_SIZE = 500
+
+
+def queryset_in_batches(queryset):
+    """Slice a queryset into batches.
+
+    Input queryset should be sorted be pk.
+    """
+    start_pk = 0
+
+    while True:
+        qs = queryset.filter(pk__gt=start_pk)[:BATCH_SIZE]
+        pks = list(qs.values_list("pk", flat=True))
+
+        if not pks:
+            break
+
+        yield pks
+
+        start_pk = pks[-1]
+
+
+def set_order_line_base_prices(apps, schema_editor):
+    def send_order_updated(sender=None, **kwargs):
+        # The `post_migrate`` signal is sent once for every app migrated
+        # we should execute it only once after update `order` module.
+        if isinstance(sender, OrderAppConfig):
+            from saleor.order.models import Order
+
+            # We need to import Order in this way because when we import it by
+            # `Order = apps.get_model("order", "Order")` models doesn't contains
+            # model methods which are required by `order_updated`
+
+            manager = get_plugins_manager()
+
+            order_ids = kwargs.get("updated_orders_pks")
+            for order in Order.objects.filter(id__in=order_ids):
+                manager.order_updated(order)
```

Comment:
if this touches a high number of orders, it would probably kill us generating payloads for each order for webhooks as that generation is synchronous and it will take a lot of time